src/Alert/Alert.js

```diff
@@ -24,21 +24,27 @@ const Alert = ({ children, onDismiss, type }) => {
   return (
     <div className={alertClass}>
       {onDismiss &&
-        <button type='button' className='close' aria-hidden='true' onClick={onDismiss}>
-          <span className='pficon pficon-close' />
+        <button
+          type="button"
```

Comment:
hey @jtomasek, the only distinction i'm seeing here in my VS Code editor is my old standardjs plugin now complains about double quote in the JSX attributes. I'm fine w/ disabling this if we prefer to use double quotes now. Is this consistent w/ your eslint config?